base/reflection.jl

```diff
@@ -450,6 +457,19 @@ function functionloc(f)
     functionloc(first(mt))
 end
 
+"""
+    function_module(f) -> Module
+
+Determine the module containing the (first) definition of a generic
+function.
+"""
+function_module(f) = datatype_module(f)
+
+"""
+  function_module(f::Function, types) -> Module
```

Comment:
Needs a four space indent for markdown to parse it as a code block. (That's where the `genstdlib.jl` warning is coming from I think.)